### PR TITLE
Feat(makefile): add rule to skip integration tests

### PR DESCRIPTION
**Pull request by GeorgeSittas**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,10 +12,15 @@ install-pre-commit:
 test:
 	python -m unittest
 
-style:
+test-unit:
```
Comment (tobymao) [inline Makefile:15]:
```suggestion
unit:
```

File: Makefile [hunk 2]
```diff
@@ -12,10 +12,15 @@ install-pre-commit:
...
+	SKIP_INTEGRATION=1 python -m unittest
+
+lint:
```
Comment (tobymao) [inline Makefile:18]: revert it back to style

File: Makefile [hunk 3]
```diff
@@ -12,11 +12,16 @@ install-pre-commit:
 test:
 	python -m unittest
 
+test-unit:
```
Comment (tobymao) [inline Makefile:15]: can you change this one to unit

File: Makefile [hunk 4]
```diff
@@ -12,11 +12,16 @@ install-pre-commit:
...
+	SKIP_INTEGRATION=1 python -m unittest
+
 style:
 	pre-commit run --all-files
 
 check: style test
 
+unit: style test-unit
```
Comment (tobymao) [inline Makefile:23]: we don't need this one

Comment (GeorgeSittas) [reply Makefile:23]: gotcha getting rid of it in a sec

File: Makefile (hunk 3 above)
Comment (GeorgeSittas) [reply Makefile]: Do we want linter checks too, or just keep it simple?

**Pull request merged**